File: 0828/1002.py
iteration_count = int(input())

result_list = []
for _ in range(iteration_count):
    input_list = input().split()
    x1, y1, r1 = input_list[0:3]
    x2, y2, r2 = input_list[3:6]

    distance = ((int(x1) - int(x2)) ** 2 + (int(y1) - int(y2)) ** 2) ** 0.5
    r_sum = int(r1) + int(r2)
    r_diff = abs(int(r1) - int(r2))

    if r_diff < distance < r_sum:
        result_list.append(2)
    elif distance == r_sum or (distance == r_diff and distance != 0):
        result_list.append(1)
    elif distance == 0 and r1 == r2:
        result_list.append(-1)
    else:
        result_list.append(0)

for result in result_list:
    print(result)

File: 0828/1002.go
package main

import (
	"bufio"
	"fmt"
	"math"
	"os"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	iterationCount, _ := strconv.Atoi(scanner.Text())

	results := make([]int, 0, iterationCount)
	for i := 0; i < iterationCount; i++ {
		scanner.Scan()
		inputList := strings.Fields(scanner.Text())
		x1, _ := strconv.Atoi(inputList[0])
		y1, _ := strconv.Atoi(inputList[1])
		r1, _ := strconv.Atoi(inputList[2])
		x2, _ := strconv.Atoi(inputList[3])
		y2, _ := strconv.Atoi(inputList[4])
		r2, _ := strconv.Atoi(inputList[5])

		distance := math.Hypot(float64(x1-x2), float64(y1-y2))
		rSum := float64(r1 + r2)
		rDiff := math.Abs(float64(r1 - r2))

		if rDiff < distance && distance < rSum {
			results = append(results, 2)
		} else if distance == rSum || (distance == rDiff && distance != 0) {
			results = append(results, 1)
		} else if distance == 0 && r1 == r2 {
			results = append(results, -1)
		} else {
			results = append(results, 0)
		}
	}

	for _, result := range results {
		fmt.Println(result)
	}
}